src/components/CarouselCard/index.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable no-script-url,max-len */
```

Comment:
when an indicator is clicked or key left or right are pressed when focused the indicator the animation must stop